epoch/api/channels_api_usage.md

```diff
@@ -518,3 +521,432 @@ details:
+
+* channel is opened on-chain
+* channel is not in a _closing_ state but not yet closed - no `channel_close_solo` had been
+  included in a block yet
+
+Any participant in the channel can post a `channel_close_solo` transaction. In
+the scope of this description we will call the one that posts the transaction
+_solo closer_.
+The transaction has the following structure:
+
+  | Name | Type | Description |
+  | ---- | ---- | ----------- |
+  | channel id | string | ID of the channel to close |
+  | from | string | solo closer's public key |
+  | payload | binary | closing payload |
+  | poi | binary | closing proof of inclusion |
+  | ttl | integer | maximum block height to include the transaction |
+  | fee | integer | fee to be paid to the miner |
+  | nonce | integer | solo closer's nonce |
+
+`payload` and `poi` are validated as [described above](#payload-and-proof-of-inclusion)
+
+#### Slash on-chain transaction
+After the channel is already in a _closing_ state, both participants can provide
+a newer state via the `channel_slash` transaction. Preconditions for
+the `channel_slash` to be valid are:
+
+* channel is opened on-chain
+* channel is still in a _closing_ state - no `channel_settle` had been
+  included in a block yet
+
+Any participant in the channel can post a `channel_slash` transaction. In
+the scope of this description we will call the one that posts the transaction
+_slasher_.
+The transaction has the following structure:
+
+  | Name | Type | Description |
+  | ---- | ---- | ----------- |
+  | channel id | string | ID of the channel to slash |
+  | from | string | slasher's public key |
+  | payload | binary | slashing payload |
+  | poi | binary | slashing proof of inclusion |
+  | ttl | integer | maximum block height to include the transaction |
+  | fee | integer | fee to be paid to the miner |
+  | nonce | integer | slasher's nonce |
+
+`payload` and `poi` are validated as [described above](#payload-and-proof-of-inclusion)
+
+#### Settle on-chain transaction
+After the `channel_close_solo` and all the `channel_slash` transactions,
+it is time to finally close the channel. One of the participants posts a
+`channel_settle` transaction that enforces closing of the channel. This
+happens according to the latest channel state that was sent on-chain. The
+`channel_settle` just finalizes the channel closing with the last received
+state, redistributes tokens to participants and closes the channel. No further
+disputes are possible after that.
+
+
+In order to give parties time to slash a closing channel and update its state
+with a newer one, there is a timeframe only after which the `channel_settle`
+can be posted. This is measured in blockes mined on top of the last received
+transaction for that channel (either `channel_close_solo` or `channel_slash`).
+The amount itself is prenegotiated before opening the channel and is part of
+the `channel_create` transaction - it is the value of `lock_period`. Under no
```

Comment:
part of the `channel_create` transaction and any subsequent deposit or withdrawal transactions.
"it is the value of `lock_period`" – what does that mean in this context? It cannot be changed during the `lock_period`?

Reply:
No Settle can happen before the lock_period passes. Also it can not be modified in deposits and withdrawals